gradebookng/tool/src/webapp/scripts/gradebook-gbgrade-table.js

```diff
@@ -924,7 +924,12 @@ GbGradeTable.renderTable = function (elementId, tableData) {
 
   GbGradeTable.instance.on("cellEdited", function(cell) {
     const oldScore = cell.getOldValue();
-    const newScore = cell.getValue();
+    let newScore = cell.getValue();
```

Comment:
I don't want to block anything, as this is already tested, just wondering if it could just be `const newScore = cell.getValue() ?? "";

Reply:
I can test again if there are some changes ;)

Reply:
its a good idea @stetsche , i will test it. thanks

Reply:
it doesn't work, it takes undefined when backspace is pressed.